docs/usage.ipynb

```diff
@@ -159,16 +200,84 @@
     }
    },
    "source": [
-    "-----\n",
-    "See `examples/qasm_demo.py` for more examples\n"
+    "Note that the above output vector represents the state before the measurement due to the optimized circuit, not the original one, which would have a plus sign instead of a minus. `IQMDevice.simplify_circuit` has eliminated a `ZPowGate` which has no effect on the measurement.\n",
```

Comment:
```suggestion
    "Note that the above output vector represents the state before the measurement in the optimized circuit, not the original one, which would have a plus sign instead of a minus. `IQMDevice.simplify_circuit` has eliminated a `ZPowGate` which has no effect on the measurement.\n",
```

Reply:
If I got what you mean there, I think it reads more clearly like this.